addons/web/static/src/core/utils/timing.js

```diff
@@ -3,6 +3,26 @@
 import { browser } from "@web/core/browser/browser";
 import { onWillUnmount, useComponent } from "@odoo/owl";
 
+/**
+ * Creates a batched version of a callback so that all calls to it in the same
+ * microtick will only call the original callback once.
```

Comment:
```suggestion
 * Creates a batched version of a callback so that all calls to it in the same
 * time frame will only call the original callback once.
```